src/Ads/AdsRecommendationsService.php

```diff
@@ -68,6 +69,17 @@ public function get_recommendations( string $type = '', int $id = 0 ): array {
 
 		if ( $id ) {
 			$query->where( 'recommendation_id', $id );
+		} else {
+			// Only return recommendations from the past 7 days.
+			$query->where( 'recommendation_last_synced', gmdate( 'Y-m-d H:i:s', time() - ( 7 * DAY_IN_SECONDS ) ), '>' );
```

Comment:
I think it's ok to include this as an extra check, but in reality, the recurring job that refreshes the recommendations every 7 days should remove all old recommendations and replace them with new ones, so this should still return all of the recommendations that were fetched from the Ads API